packages/config/src/options.js

```diff
@@ -321,16 +321,19 @@ export function getNuxtConfig(_options) {
 
   // include SFCs in node_modules
   options.build.transpile = [].concat(options.build.transpile || [])
-    .map(module => (module instanceof RegExp)
-      ? module
-      : new RegExp(
-        escapeRegExp(
-          path.normalize(
-            module.replace(/\\/g, '/')
-          )
-        )
-      )
-    )
+  const include = []
+  const whitelist = []
+  for (const module of options.build.transpile) {
```

Comment:
Not related to this PR, but module is a CJS reserved keyword :D

Reply:
Shouldnt eslint catch this?

Reply:
It also happened somewhere else in project, let me change this for now.

Reply:
@aldarund ESLint is not always smart! Try`import path from 'path'` and then modify `path` variable 😆